cockpit/src/subscriptions-view.jsx

```diff
@@ -177,128 +212,87 @@ class SubscriptionStatus extends React.Component {
             this.props.unregister();
         e.stopPropagation();
     }
+
     render() {
-        let errorMessage;
-        if (this.props.error) {
-            errorMessage = (
-                <DismissableError dismissError={this.props.dismissError}
-                                  severity={this.props.error.severity}>
-                    {this.props.error.msg.toString()}
-                </DismissableError>
+        let syspurpose = null;
+        const p = this.props.syspurpose;
+        if (p["service_level_agreement"] || p["usage"] || p["role"] || p["addons"]) {
+            syspurpose = (
+                <Card id="syspurpose" key="syspurpose" className="ct-card-info">
+                    <CardHeader>
+                        <CardTitle><Text component={TextVariants.h2}>{_("System purpose")}</Text></CardTitle>
+                    </CardHeader>
+                    <CardBody>
+                        <DescriptionList isHorizontal>
+                            <DescriptionListGroup>
+                                <DescriptionListTerm>{_("Status")}</DescriptionListTerm>
```

Comment:
Can you please display "Status" little bit different? Currently it looks like that "Status" is another system purpose attribute, which is not true.

Reply:
Right, I was not sure about this one. What are the possible values? And what importance does it have? For example, can it be badge in the card title?

Reply:
Value displayed in this Status is parsed from JSON document returned from candlepin server. So I believe that possible values are defined on candlepin server here: https://github.com/candlepin/candlepin/blob/master/server/src/main/java/org/candlepin/policy/SystemPurposeComplianceStatus.java#L35. I can see there 4 values + "undefined", when we are not able to connect to candlepin server. I don't think that it is possible to visualize with simple badge. I'm not UX designer. So I don't know what is the best way how to communicate this value to users.

Reply:
Ok, so I gave this though and came up with this:
![purposenew](https://user-images.githubusercontent.com/12330670/118281874-fbc1a780-b4cd-11eb-8ac6-a4689a963957.png)

WDYT?

Also for the record, if neither of the fields is set up, this card is not shown (as there is nothing to show nor do - if once setting of purpose would be implemented then it can be always shown with empty state)
![nopurp ose](https://user-images.githubusercontent.com/12330670/118282029-227fde00-b4ce-11eb-8c8f-b13a6bcb6ff3.png)